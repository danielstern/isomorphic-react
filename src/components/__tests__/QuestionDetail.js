describe('The Question Detail Component', () => {
  it('Should not regress', () => {

  });
});