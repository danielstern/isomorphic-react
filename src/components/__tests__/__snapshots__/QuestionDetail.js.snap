// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Question Detail Component The display element Should not regress 1`] = `
<div>
  <h3
    className="mb-2"
  >
    THANKS!
  </h3>
  <div>
    <div
      className="mb-3"
    >
      <div>
        <code>
          hitchhiking
        </code>
      </div>
    </div>
    <div>
      <p>
        43
      </p>
    </div>
    <div>
      0
       Answers
    </div>
  </div>
</div>
`;
